lpr.WebAPI/Controllers/OrganisationController.cs

```diff
@@ -37,6 +37,21 @@ public class OrganisationController : ControllerBase {
       return StatusCode(200, org);
     }
 
+    [HttpGet("GetPackagesPaginated/{page}/{amount}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<IActionResult> GetOrganisationsPaginated(int page,
+                                                               int amount) {
+      try {
+        List<Organisation> output =
+            await _organisationService.GetOrganisationsPaginatedAsync(page,
+                                                                      amount);
+        return StatusCode(200, output);
+      } catch (Exception ex) {
+        return StatusCode(500, new ErrorMessage(ex.Message));
```

Comment:
Please don't return the error message. This can be exploited in the future. A 500 with a get is more than enough